lib/fastlane/plugin/firebase_test_lab/options.rb

```diff
@@ -122,6 +122,11 @@ def self.available_options
                                        description: "Set to true if you want to rerun test sute when failed. Default: nil",
                                        is_string: false,
                                        optional: true,
+                                       default_value: nil),
+          FastlaneCore::ConfigItem.new(key: :print_successful_test,
+                                       description: "Set to true all successful tests will be printed. Default: nil",
+                                       is_string: false,
+                                       optional: true,
                                        default_value: nil)
```

Comment:
Maybe we could make it a specific Boolean value, like we do with skip_validation and still it's optional by default:
`          FastlaneCore::ConfigItem.new(key: :skip_validation,
                                       description: "Do not validate the app before uploading",
                                       default_value: false,
                                       type: Fastlane::Boolean),
`